snapcraft.yaml

```diff
@@ -197,6 +193,23 @@ parts:
         cp target/release/dump_syms $SNAPCRAFT_STAGE/usr/bin/
       fi
 
+  hunspell:
+    plugin: nil
+    override-build: |
+      set -eu
+      apt download $(apt-cache search '^hunspell-.*$' |
+        awk '!/myspell|dbgsym|tools|transitional|dependency/{printf "%s ", $1}')
+      find . -name "*.deb" -exec dpkg-deb -x {} "$SNAPCRAFT_PART_INSTALL" \;
+    filesets:
+      copyright: [usr/share/doc/*/copyright]
+      hunspell-dictionaries: [usr/share/hunspell]
+    stage:
+      - $copyright
+      - $hunspell-dictionaries
+    prime:
+      - $copyright
```

Comment:
Given that this is a temporary solution, and that snaps are not bound to the Debian policy of shipping copyright files, I think we should not ship `$copyright`. This would save 8MB in the uncompressed snap.